php/EE/Migration/Executor.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace EE\Migration;
+
+use \EE;
+use \EE\Utils;
+use Symfony\Component\Finder\Finder;
+
+class Executor {
+
+    const MIGRATION_PATH = '/Users/kirtan/code/easyengine/easyengine/migrations/';
```

Comment:
@kirtangajjar You seem to have accidentally committed hardcoded path to your migrations directory

Reply:
Whoops yep... Fixing it.

Reply:
@mbtamuli fixed :)